pages/common/blender.md

```diff
@@ -0,0 +1,32 @@
+# blender
+
+> Command-line interface to the Blender 3D computer graphics application.
+> Arguments are executed in the order they are given.
+
+- Render all frames of an animation in the background (without UI):
+
+`blender -b {{file.blend}} -a`
+
+- Render an animation to a specific output path relative to the .blend file:
+
+`blender -b {{file.blend}} -o //{{render/frame_###.png}} -a`
+
+- Render the 10th frame of an animation as a single image:
```

Comment:
Oh, good point. I suppose all these invocations should have output paths. The default is saving to /tmp, which is not intuitive IMO.